replicator/src/core.rs

```diff
@@ -11,15 +11,30 @@ use etl::types::PipelineId;
 use etl_destinations::bigquery::{BigQueryDestination, install_crypto_provider_for_bigquery};
 use secrecy::ExposeSecret;
 use tokio::signal::unix::{SignalKind, signal};
-use tracing::{debug, info, warn};
+use tracing::{debug, info, instrument, warn};
 
 use crate::config::load_replicator_config;
 use crate::migrations::migrate_state_store;
 
 pub async fn start_replicator() -> anyhow::Result<()> {
-    info!("starting replicator service");
-
     let replicator_config = load_replicator_config()?;
+    let project_ref = replicator_config
+        .supabase
+        .as_ref()
+        .map(|s| s.project_ref.clone())
+        .unwrap_or_default();
+    start_with_config(replicator_config, &project_ref).await
+}
+
+// The name of the field emitted in the instrumented span should be `project`
+// not `project_ref` because the vector config in the etl-k8s project expects
+// this. These should be kept in sync if changed in future.
+#[instrument(skip(replicator_config, project_ref), fields(project = project_ref))]
+async fn start_with_config(
```

Comment:
nit:
```suggestion
async fn start_replicator_with_config(
```

Reply:
Sure, but our project is open source and our logging impl is generic, so we should also take into consideration that people might be running ETL in different setups.

Reply:
Why would someone using the open source version mix logs from the API and the replicator crate into the same system? These are two separate binaries which emit separate logs.

Reply:
My point was more generic, that we should not think only at our infra when developing ETL. For this case it's fine.